app/views/revision_feedback/index.html.haml

```diff
@@ -1,6 +1,12 @@
 %section
   %h1
     Suggestions
+  %h4
+    Rating
+  %p
+    = @rating
+  %h4
+    More Suggestions
```

Comment:
Any reason to keep the html version of this, since we're only using the json one?